### PR TITLE
Add documentation for lib/rescuecdstep.pm

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/57332

### CODE REVIEW

File: lib/rescuecdstep.pm [hunk 1]
```diff
@@ -6,6 +12,11 @@ use warnings;
 
 # Base class for all RESCUECD tests
 
+=head2 test_flags
+
```
Comment (asdil12) [inline lib/rescuecdstep.pm]:
```suggestion

 test_flags();

```

File: lib/rescuecdstep.pm [hunk 2]
```diff
@@ -6,6 +12,11 @@ use warnings;
...
+Return test flag fatal => 1
```
Comment (asdil12) [inline lib/rescuecdstep.pm]:
```suggestion
rescuecdstep tests are fatal by default (eg. the testsuite will stop immediately if this tests fail).
```

Comment (Zaoliang) [reply lib/rescuecdstep.pm]: thanks

Comment (SergioAtSUSE) [inline lib/rescuecdstep.pm]: Something may have been wrong with this. The conversation is shown as resolved, but I cannot see the changes.

Comment (Zaoliang) [reply lib/rescuecdstep.pm]: ah, change was not there.

Comment (Zaoliang) [reply lib/rescuecdstep.pm]: it should be okay now.

Comment (asdil12) [reply lib/rescuecdstep.pm]: sorry, but I still can't see the changes. There is also no indication on github, that you pushed again.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8533.patch should contain the changes

**Pull request merged**